pulsar-client-cpp/lib/MultiTopicsConsumerImpl.cc

```diff
@@ -712,7 +738,19 @@ void MultiTopicsConsumerImpl::seekAsync(const MessageId& msgId, ResultCallback c
 }
 
 void MultiTopicsConsumerImpl::seekAsync(uint64_t timestamp, ResultCallback callback) {
-    callback(ResultOperationNotSupported);
+    Lock stateLock(mutex_);
+    if (state_ != Ready) {
+        stateLock.unlock();
+        callback(ResultAlreadyClosed);
+        return;
+    }
+    // consumers_ could only be modified when state_ is Ready, so we needn't lock consumersMutex_ here
```

Comment:
Please rebase master and remove stateLock.

Reply:
Thanks, fixed.